README.md

```diff
@@ -83,3 +81,75 @@ Generally not applicable. If there are spoiler concerns in documentation or guid
 1. When possible, spoilers should be grouped together into a single section per page/topic. We should not use inline spoiler obfuscation that occurs in random short chunks throughout a page. They should instead exist in their own, collapsible section.
 1. When possible, spoilers should be grouped together into as few of these blocks as possible per page. Small blocks, or other non-standard inline obfuscation, should be avoided.
 1. In cases where the vast majority of the content of a given page would be classified as spoilers, and hiding so much of the content would be impractical or otherwise undesirable, it is sufficient to include an overt message warning readers of the spoilers ahead instead of hiding them. This warning should exist either at the top of the page or immediately after a small introductory section that is itself free of spoilers.
+
+### Adding Spoilers
+1. Spoilers should be added by following the following example:
+```html
+<details>
+  <summary>Lorem Ipsum</summary>
+  Dolor icecream :)
+</details>
+```
+
+### General
+1. Empty articles or placeholder articles should not be created. Only create articles with the intention to provide a majority of the info. 
+1. All articles should pertain to hackmud, its lore, or its mechanics. 
+1. Articles should not be created pertaining to the developers, players, or any other real life person. 
+1. All articles relating to mechanics should be mechanics that are currently in game and not historic or removed info. 
+1. All articles are markdown format, using the mdx file extension. 
+1. Any game text content/output should not be correct for formatting, spelling, or grammar. If information does not pertain to the article, omitting the lines and replacing them with “<lines omitted>” is acceptable.
+1. When in doubt, infer appropriate styling/content from previously written articles. If assumptions are still made, list these in a separate Pull Request so that they can be addressed for content guide improvements.
```

Comment:
> separate Pull Request

Should assumptions be listed in the article PR first? That way, reviewers can identify if the issue is the CG, and if this blocks the article PR until the issue is resolved (e.g. missing guideline that needs further discussion). I think this would save time, and would be more consistent with us discovering an issue with the article PR and realising the problem is the CG (e.g. "Thank you for the contribution but this article is not in English" "actually CG says Welsh is allowed" "oh ok, we will amend the CG then. that was a typo")

Reply:
I'm don't remember if it was in this exact context, but there was some discussion about an unwillingness to require that the person who wrote the article (and thus the PR) also be the one responsible for fixing it (i.e. by blocking the PR), since technically they correctly followed the existing Content Guide at the time of their PR.

Having it be separate introduces more process, but it also opens it up for some other volunteer (or the same person if they're interested) to patch up the article once problems with the Content Guide have been resolved.

Reply:
On the topic of just issue discovery, I imagine it would come in whatever PR and be asked or proposed to make an issue for it. However, if you have an issue in mind while making your original PR, it's best to create a second PR... is what I was thinking

Reply:
Thanks for your thoughts and giving me the added context. I think I'm on the same page, so although I haven't figured out what phrasing I'd suggest instead, I'm happy to move forward with what we have.